.github/CODEOWNERS

```diff
@@ -15,18 +15,23 @@
 *                               @glotaran/admins
 
 # tooling
-/.github /.*                    @glotaran/admins @glotaran/maintainers
-/*.y*ml /*.ini /*.toml /*.txt   @glotaran/admins @glotaran/maintainers
-LICENSE                         @pyglotaran_creators
+/.github                        @glotaran/admins @glotaran/maintainers
+/.*                             @glotaran/admins @glotaran/maintainers
+/*.y*ml                         @glotaran/admins @glotaran/maintainers
+/*.ini                          @glotaran/admins @glotaran/maintainers
+/*.toml                         @glotaran/admins @glotaran/maintainers
+/*.txt                          @glotaran/admins @glotaran/maintainers
+LICENSE                         @glotaran/pyglotaran_creators
 
 # docs
-/docs *.rst *.md                @glotaran/maintainers @pyglotaran_creators
+/docs/**/*.rst                  @glotaran/maintainers @glotaran/pyglotaran_creators
+# /docs/**/*.md                   @glotaran/maintainers @glotaran/pyglotaran_creators
```

Comment:
Why is this left in a a comment?

Reply:
Just for when we add `*.md` files to the docs.
With the [myst-parser](https://myst-parser.readthedocs.io/en/latest/using/syntax.html) you get the power of `*.rst` in `*.md` 😉